ganttproject/src/main/java/net/sourceforge/ganttproject/calendar/MultiDatePicker.kt

```diff
@@ -0,0 +1,96 @@
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with GanttProject.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+
+package net.sourceforge.ganttproject.calendar
+
+import com.sun.javafx.scene.control.DatePickerContent
+import javafx.event.EventHandler
+import javafx.scene.Node
+import javafx.scene.control.DateCell
+import javafx.scene.control.DatePicker
+import javafx.scene.control.skin.DatePickerSkin
+import javafx.scene.input.MouseButton
+import javafx.scene.input.MouseEvent
+import javafx.util.Callback
+import java.time.LocalDate
+
+
+class MultiDatePicker : DatePicker() {
+    private val selectedDatesList = mutableListOf<LocalDate>()
+    private var lastSelectedDate: LocalDate? = null
+
+  val selectedDates: List<LocalDate> get() = selectedDatesList.toList()
+  val popupContent: Node
+    get() = (skin as DatePickerSkin).popupContent
+
+  init {
+        skin = DatePickerSkin(this)
+        val mouseClickedEventHandler = EventHandler { clickEvent: MouseEvent ->
+            if (clickEvent.button == MouseButton.PRIMARY) {
+              lastSelectedDate?.let {
+                    selectedDatesList.clear()
+                    var startDate: LocalDate
+                    val endDate: LocalDate
+                    if (value.isAfter(it)) {
+                        startDate = it
+                        endDate = value
+                    } else {
+                        startDate = value
+                        endDate = it
+                    }
+                    do {
+                        selectedDatesList.add(startDate)
+                        startDate = startDate.plusDays(1)
+                    } while (!startDate.isAfter(endDate))
+                } ?: run {
+                  selectedDatesList.add(value)
+                }
+                lastSelectedDate = value
+            }
+            val datePickerContent = popupContent as DatePickerContent
+            datePickerContent.updateDayCells()
+            clickEvent.consume()
+        }
+        dayCellFactory = Callback {
+          object : DateCell() {
+                override fun updateItem(item: LocalDate?, empty: Boolean) {
+                    super.updateItem(item, empty)
+                    //...
+                    if (item != null && !empty) {
+                        //...
+                        addEventHandler(MouseEvent.MOUSE_CLICKED, mouseClickedEventHandler)
+                    } else {
+                        //...
+                        removeEventHandler(MouseEvent.MOUSE_CLICKED, mouseClickedEventHandler)
+                    }
+                    style = if (selectedDatesList.isNotEmpty() && selectedDatesList.contains(item)) {
+                        if (item == selectedDatesList.toTypedArray()[0] || item == selectedDatesList.toTypedArray()[selectedDatesList.size - 1]) {
+                            "-fx-background-color: rgba(3, 169, 1, 0.7);"
```

Comment:
I suggest using CSS classes instead of hardcoded styles. I created an SCSS file for MultiDatePicker and defined a couple of test styles (uncomment them to see the effect on the buttons)

Reply:
Two color styles are added to MultiDatePicker css